x/ccv/consumer/keeper/params_test.go

```diff
@@ -7,12 +7,12 @@ import (
 
 func (suite *KeeperTestSuite) TestParams() {
 	// suite setup initializes genesis
-	expParams := types.NewParams(true, 1000, "", "") // these are the default params
+	expParams := types.NewParams(true, 1000, "", "", false) // these are the default params
```

Comment:
default should be lockUnbondingOnTimeout=true right?

https://github.com/cosmos/ibc/issues/669#issuecomment-1087320240

I know this is a test so maybe it's just the comment that's misleading